tasks/setup.rake

```diff
@@ -53,6 +53,15 @@ namespace "setup" do
     version = Setup.get_package_version
     good("Published egg version #{version} to index #{Project[:INDEX_URL]}")
   end
+
+  desc "Publish the egg to the artifactory, make sure a there is a .pypirc"
+  task :artifactory => ["virtualenv:verify", "version:inject_git", "version:inject_build_number"] do
+    Dir.mktmpdir do |tmpdir|
+      Setup.setup %|sdist -d "#{tmpdir}" upload -r local|
```

Comment:
what command is this running? is this calling the jfrog cli?

Reply:
This is running the setup.py script and is not using jfrog cli
i did it as suggested [here](https://www.jfrog.com/confluence/display/RTF/PyPI+Repositories) and tested it with astrojenkins

Reply:
https://mhcastrojenkins-build.aws.wgen.net/jenkins/job/asiaq-BUILD/517/console